orc8r/cloud/go/services/configurator/storage/sql_entity_load_helpers.go

```diff
@@ -51,12 +53,16 @@ func (store *sqlConfiguratorStorage) loadFromEntitiesTable(networkID string, fil
 	return entsByPk, nil
 }
 
-func (store *sqlConfiguratorStorage) getLoadEntitiesSelectBuilder(networkID string, filter EntityLoadFilter, criteria EntityLoadCriteria) sq.SelectBuilder {
+func (store *sqlConfiguratorStorage) getLoadEntitiesSelectBuilder(networkID string, filter EntityLoadFilter, criteria EntityLoadCriteria) (sq.SelectBuilder, error) {
 	// SELECT ent.pk, ent.key, ent.type, ent.physical_id, ent.version, graph.graph_id, ent.name, ent.description, ent.config,
 	// [[ acl.id, acl.scope, acl.permission, acl.type, acl.id_filter, acl.version ]]
 	// FROM cfg_entities AS ent
 	// [[ LEFT JOIN cfg_acls AS acl ON acl.entity_pk = ent.pk ]]
 	// [[ WHERE (ent.network_id = $1 AND ent.key = $2 AND ent.type = $3) OR (ent.network_id ...) ... ]]
+	// [[ ORDER BY ent.key ASC LIMIT page_size ]]
+	if err := validatePaginatedLoadParameters(filter, criteria); err != nil {
```

Comment:
Hoist this validation up to the caller -- there's only one call site, and not returning an error feels cleaner if we can help it

Generally speaking, I like to validate parameters as far toward the edge of the interface as possible. That way all internal code can follow the "if it exists then it's a valid value" rule, which makes for simpler implementations and easier to reason about